src/legacy/core_plugins/kibana/public/dev_tools/index.ts

```diff
@@ -17,18 +17,13 @@
  * under the License.
  */
 
-import { npSetup, npStart } from 'ui/new_platform';
+// make sure all dev tools are loaded and registered.
+import 'uiExports/devTools';
 
-import { DevToolsPlugin } from './plugin';
-import { localApplicationService } from '../local_application_service';
+import { npStart } from 'ui/new_platform';
 
-const instance = new DevToolsPlugin();
-
-instance.setup(npSetup.core, {
-  __LEGACY: {
-    localApplicationService,
-  },
-});
-instance.start(npStart.core, {
-  newPlatformDevTools: npStart.plugins.devTools,
-});
+if (npStart.plugins.devTools.getSortedDevTools().length === 0) {
```

Comment:
@restrry qq: What's the convention for plugin naming? There is `feature_catalogue`, but `uiActions`

Reply:
all legacy plugins use `snake_case` for naming https://github.com/elastic/kibana/tree/master/src/legacy/core_plugins

Reply:
@streamich do you want me to create an issue to fix that for `ui_actions`?

Reply:
@restrry Ah, maybe I wasn't clear. I was talking about the `id` of the plugin. The `ui_actions`/`uiActions` plugin uses snake case for the directory, but camel case for the id.

Reply:
@flash1293 I got it. I didn't find `id` in camelCase within  https://github.com/elastic/kibana/tree/master/src/legacy/core_plugins

Reply:
@flash1293 I got it. I didn't find `name` in camelCase within  https://github.com/elastic/kibana/tree/master/src/legacy/core_plugins
In LP `name` is `id` equivalent https://github.com/restrry/kibana/blob/41b49358e6626809d0ed479c53aa711bb430bbbe/src/legacy/plugin_discovery/plugin_spec/plugin_spec.js#L32

Reply:
https://github.com/elastic/kibana/issues/52190